CRM/Contribute/Form/Contribution.php

```diff
@@ -1252,7 +1260,10 @@ protected function processCreditCard($submittedValues, $lineItem, $contactID) {
       // NOTE - I expect this is obsolete.
       $payment = Civi\Payment\System::singleton()->getByProcessor($this->_paymentProcessor);
       try {
-        $statuses = CRM_Contribute_BAO_Contribution::buildOptions('contribution_status_id');
+        $statuses = CRM_Core_PseudoConstant::get('CRM_Contribute_DAO_Contribution',
```

Comment:
why did you make this change? My understanding is that @colemanw intends the buildOptions as the preferred method???